src/linea/actions/lineaEstimateFeesPerGas.ts

```diff
@@ -0,0 +1,157 @@
+import {
+  http,
+  type Block,
+  type Chain,
+  type ChainEstimateFeesPerGasFnParameters,
+  type Client,
+  Eip1559FeesNotSupportedError,
+  type EstimateGasParameters,
+  type Transport,
+  createPublicClient,
+} from '../../index.js'
+import { getAction } from '../../utils/getAction.js'
+import { linea, lineaTestnet } from '../chains.js'
+import type {
+  LineaEstimateFeesPerGasReturnType,
+  LineaEstimateGasReturnType,
+} from '../types/fee.js'
+import { lineaEstimateGas } from './lineaEstimateGas.js'
+
+const BASE_FEE_PER_GAS_MARGIN = 1.35
+
+/**
+ * Returns an estimate for the fees per gas (in wei) for a
+ * transaction to be likely included in the next block.
+ *
+ * @param client - Client to use
+ * @param parameters - {@link ChainEstimateFeesPerGasFnParameters}
+ * @returns An estimate (in wei) for the base fee per gas, the priority fee per gas and the gas limit. {@link LineaEstimateFeesPerGasReturnType}
+ *
+ * @example
+ * import { createPublicClient, http } from 'viem'
+ * import { mainnet } from 'viem/chains'
+ * import { estimateFeesPerGas } from 'viem/actions'
+ *
+ * const client = createPublicClient({
+ *   chain: mainnet,
+ *   transport: http(),
+ * })
+ *
+ * const estimatedFeesPerGas = await lineaEstimateFeesPerGas({
+ *   client,
+ *   request: requestExample,
+ * } as any);
+ * // { maxFeePerGas: ..., maxPriorityFeePerGas: ..., gasLimit: ... }
+ */
+export async function lineaEstimateFeesPerGas({
+  multiply,
+  block,
+  client,
+  request,
+  type,
+}: ChainEstimateFeesPerGasFnParameters): Promise<LineaEstimateFeesPerGasReturnType> {
+  if (!request) {
+    throw new Error('Request parameters are missing')
+  }
+
+  let lineaEstimateGasResponse: LineaEstimateGasReturnType
+  try {
+    lineaEstimateGasResponse = await getAction(
+      client,
+      lineaEstimateGas,
+      'lineaEstimateGas',
+    )({
+      data: request.data,
+      to: request.to,
+      ...request,
+    } as unknown as EstimateGasParameters)
+  } catch (_err) {
+    try {
+      // Try with the public endpoint
+      const publicClient = createPublicClient({
+        chain: client.chain.id === linea.id ? linea : lineaTestnet,
+        transport: http(),
+      })
+      lineaEstimateGasResponse = await getAction(
+        publicClient,
+        lineaEstimateGas,
+        'lineaEstimateGas',
+      )({
+        data: request.data,
+        to: request.to,
+        ...request,
+      } as unknown as EstimateGasParameters)
+    } catch (_err) {
+      // Return the result using the standard eth_gasPrice method
+      return lineaEstimateFeesPerGasFallback({
+        multiply,
+        block,
+        client,
+        request,
+        type,
+      })
+    }
+  }
+
+  const { baseFeePerGas, priorityFeePerGas, gasLimit } =
+    lineaEstimateGasResponse
+
+  const adjustedPriorityFeePerGas = BigInt(priorityFeePerGas)
+  const adjustedBaseFee =
+    (BigInt(baseFeePerGas) * BigInt(BASE_FEE_PER_GAS_MARGIN * 100)) /
+    BigInt(100)
```

Comment:
Can we rely on `multiply` here? This is default to `1.2`, but can be adjusted to `1.35` via [`chain.fees.baseFeeMultiplier`](https://viem.sh/docs/chains/fees#feesbasefeemultiplier)

```suggestion
  const adjustedBaseFee = multiply(BigInt(baseFeePerGas))
```

Reply:
hey @jxom we will keep the default value 1.2